core/src/http/AuthHttpClient.js

```diff
@@ -646,12 +619,12 @@ export class AuthHttpClient {
 
         const versionHeader = new VersionHeader();
         const developerHeader = new DeveloperHeader(this._developerKey);
-        const securityMetadataHeader = new SecurityMetadataHeader(this._securityMetadata);
+        const customerTrackingMetadataHeader = new CustomerTrackingMetadata(this._context.customerTrackingMetadata);
         this._interceptor = this._instance.interceptors.request.use(async request => {
             await this._authHeader.addAuthorizationHeader(this._memberId, request, this._context);
             versionHeader.addVersionHeader(request);
             developerHeader.addDeveloperHeader(request);
-            securityMetadataHeader.addSecurityMetadataHeader(request);
+            customerTrackingMetadataHeader.addCustomerTrackingData(request);
```

Comment:
I am not sure if this is the right way to do it, the new headers are just like the exisiting `onBehalfOf` or `customerInitiated` headers, please refer to their implementations.

Reply:
done